experimental/casemap/Cargo.toml

```diff
@@ -47,17 +45,22 @@ icu_casemap_data = { version = "~1.3.0", path = "data", optional = true }
 
 [features]
 default = ["compiled_data"]
-std = ["icu_collections/std", "icu_provider/std"]
```

Comment:
I believe other crates have an `std` feature even if it doesn't do anything. We might need it in the future.